fanxhe/blog
# Add jekyll gem: install and configure

![image](https://cloud.githubusercontent.com/assets/8801281/8348847/8c788a5a-1add-11e5-942d-78bd0d9a514d.png)

To test this PR, please:

```
    $ git pull origin chore/configure-jekyll
    $ bundle install
    $ cd fanxhe
    $ jekyll serve

```

After it go to http://localhost:4000 to see the jekyll blog running
